File: 1주차/1001/혜인.cpp
try :
    # 입력받기
    input_value = input().strip()

    # 입력값이 정확히 하나의 숫자인지 확인
    if len(input_value.split()) > 1:
print("입력 값이 너무 많습니다. 하나의 숫자만 입력해주세요.")
    else:
N = int(input_value)

# 입력값이 조건을 만족하는지 확인
if 1 <= N <= 9:
# 구구단 출력
for i in range(1, 10) :
    print(f"{N} * {i} = {N * i}")
else:
# 입력값이 조건에 맞지 않을 때
print("입력 값이 조건을 만족하지 않습니다. N은 1 이상 9 이하의 정수여야 합니다.")
except ValueError :
# 입력값이 정수가 아닌 경우
print("유효하지 않은 입력입니다. 정수를 입력해주세요.")

File: 3주차/2675/혜인.py
a = int(input())
for _ in range(a):
    b, c = input().split() #같은 줄에 있으므로 split으로 입력받음
    for i in range(len(c)):
        print(c[i]*int(b), end='') #한줄에 여러개 입력하려면 end= '' 필요
    print('') #한 칸 띄워줘야 다음 값 입력받을 수 있음 
